### PR TITLE
Removing scope: unused options and logic

### PR DESCRIPTION
**Pull request by atz**
The following options are removed because their values can be assumed:
- `cleanup: false`
- `compute_checksum: true`
- `new_druid_tree_format: true`
- `init_assembly_wf: true`
- `limit_n` (no limit)
- `reaccession: false`
- `resume: true`
- `show_progress: false` (irrelevant)
- `throttle_time: 0`
- `uniqify_source_ids: false`
- `validate_usage: true`

Update YAML fixtures.  Resolve setup failure of one test object.

Only PART of #133, which is amazing given that so much logic is already removed here, especially many "developer only" features.

### CODE REVIEW

File: spec/bundle_spec.rb [hunk 1]
```diff
@@ -50,14 +49,7 @@ def bundle_setup(bundle_context)
   end
 
   describe '#load_skippables' do
-    it "does nothing if resume returns false" do
```
Comment (SaravShah) [inline spec/bundle_spec.rb]: oh, im so happy this is removed

File: README.md [hunk 2]
```diff
@@ -177,14 +177,11 @@ subdirectory.
 # Normal run.  Will restart and crete a new log file, overwriting any existing log file for that project.
 bin/pre-assemble YAML_FILE
 
-# Run in resume mode, which will automatically pick up where left off based on the log file.  Passing the --resume flag overrides the actual value of resume from the YAML config.
-bin/pre-assemble YAML_FILE --resume
+# Run in limit mode (default of 200), which will automatically limit the number of items pre-assembled to 200 regardless of what is set in the YAML file.
```
[the remark this answers is not in the document]
Comment (atz) [reply README.md]: I deferred fixing 2 things in detail: `bin` directory stuff, `bin` related stuff in `README`.  Most of README will go away after the deletion of scripts it references, so I wouldn't bother w/ it here.

File: lib/pre_assembly/bundle.rb [hunk 3]
```diff
@@ -273,13 +234,10 @@ class Bundle
...
-             :resume,
              :progress_log_file,
-             :show_progress,
              :content_md_creation,
              :stageable_discovery,
              :bundle_dir,
-             :limit_n,
              :project_style,
              :project_name,
              :object_discovery,
```
[the remark this answers is not in the document]
Comment (atz) [reply lib/pre_assembly/bundle.rb]: Yeah, looks like it.  There are likely to be other follow-on cleanups, too (as we complete the rest of the ticket).  I'll add this one.

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (SaravShah) [reply README.md]: gotcha, no worries. thanks

File: lib/pre_assembly/bundle.rb (hunk 3 above)
[the remark this answers is not in the document]
Comment (atz) [reply lib/pre_assembly/bundle.rb]: @SaravShah that is getting a little bit involved, can I punt it for a follow-up?

[the remark this answers is not in the document]
Comment (SaravShah) [reply lib/pre_assembly/bundle.rb]: yup! looks good otherwise

**Comment by SaravShah** [comment]
will let one more set of 👀 review this before merge!

**Comment by atz** [comment]
rebased a couple times

**Comment by atz** [comment]
OK, one more rebase fix

**Pull request merged**